axelrod/tests/unit/test_fingerprint.py

```diff
@@ -0,0 +1,155 @@
+        af.fingerprint(turns=10, repetitions=2, step=0.5, progress_bar=False,
+                       in_memory=True)
+        edge_keys = sorted(list(af.interactions.keys()))
+        coord_keys = sorted(list(af.data.keys()))
+        self.assertEqual(af.step, 0.5)
+        self.assertEqual(af.spatial_tournament.interactions_dict,
+                         af.interactions)
+        self.assertEqual(edge_keys, self.expected_edges)
+        self.assertEqual(coord_keys, self.expected_points)
+
+    def test_serial_fingerprint(self):
+        af = AshlockFingerprint(self.strategy, self.probe)
+        data = af.fingerprint(turns=10, repetitions=2, step=0.5, progress_bar=False)
+        edge_keys = sorted(list(af.interactions.keys()))
+        coord_keys = sorted(list(data.keys()))
+        self.assertEqual(af.step, 0.5)
+        self.assertEqual(edge_keys, self.expected_edges)
+        self.assertEqual(coord_keys, self.expected_points)
+
+    @unittest.skipIf(axl.on_windows,
+                     "Parallel processing not supported on Windows")
+    def test_parallel_fingerprint(self):
+        af = AshlockFingerprint(self.strategy, self.probe)
+        af.fingerprint(turns=10, repetitions=2, step=0.5, processes=2,
+                       progress_bar=False)
+        edge_keys = sorted(list(af.interactions.keys()))
+        coord_keys = sorted(list(af.data.keys()))
+        self.assertEqual(af.step, 0.5)
+        self.assertEqual(edge_keys, self.expected_edges)
+        self.assertEqual(coord_keys, self.expected_points)
+
+    def test_generate_data(self):
+        af = AshlockFingerprint(self.strategy, self.probe)
+        edges, players = af.construct_tournament_elements(0.5)
+        spatial_tournament = axl.SpatialTournament(players,
+                                                   turns=10,
+                                                   repetitions=2,
+                                                   edges=edges)
+        results = spatial_tournament.play(progress_bar=False,
+                                          keep_interactions=True)
+        data = af.generate_data(results.interactions, self.expected_points,
+                             self.expected_edges)
+        keys = sorted(list(data.keys()))
+        values = [0 < score < 5 for score in data.values()]
+        self.assertEqual(sorted(keys), self.expected_points)
+        self.assertEqual(all(values), True)
+
+    def test_plot(self):
+        af = AshlockFingerprint(self.strategy, self.probe)
+        af.fingerprint(turns=10, repetitions=2, step=0.25, progress_bar=False)
+        p = af.plot()
+        self.assertIsInstance(p, matplotlib.pyplot.Figure)
+        q = af.plot(col_map='jet')
+        self.assertIsInstance(q, matplotlib.pyplot.Figure)
+
+    def test_actual_data_fingerprint(self):
+        axl.seed(0)  # Fingerprinting is a random process
+        test_data = OrderedDict([(Point(x=0.5, y=0.75), 2.200),
```

Comment:
Can you change this to be a dict (my mistake, should have done that on my PR).

So:

```
test_data = {Point(x=0.5, y=0.75): 2.200,...
```

Then can you remove `from collections import OrderedDict` from line 8.